src/constants/translations/en/iconsTooltip.json

```diff
@@ -0,0 +1,9 @@
+{
+    "language": "Language",
+    "messages": "Messages",
+    "favorites": "Favorites",
+    "notifications": "Notifications",
+    "account": "Account",
+    "menu": "Menu",
+    "login": "Login"
+}
```

Comment:
Please add end of line (EOL) character here.

Reply:
fixed.